src/pivotal-ui/components/grids/grids.scss

```diff
@@ -2264,3 +2264,525 @@ page](/containers.html).
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">Offset 0, Size 4</div>
+ </div>
+ <div class="fitem-xs-offset-4 fitem-xs-16 mbxl">
+  <div class="bg-neutral-10">Offset 4, Size 16</div>
+ </div>
+</div>
+```
+
+Alignment and justification:
+
+```html_example
+<div class="frow flex-justify-start-xs">
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">justify start</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+</div>
+<div class="frow flex-justify-center-xs">
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">justify center</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+</div>
+<div class="frow flex-justify-end-xs">
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">justify end</div>
+ </div>
+</div>
+
+<hr>
+
+<div class="frow flex-align-start-xs">
+ <div class="fitem-xs mbxl"><div class="bg-neutral-10 pal" style="height: 5em">Tall</div></div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">align start</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+</div>
+<div class="frow flex-align-center-xs">
+ <div class="fitem-xs mbxl"><div class="bg-neutral-10 pal" style="height: 5em">Tall</div></div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">align center</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+</div>
+<div class="frow flex-align-end-xs">
+ <div class="fitem-xs mbxl"><div class="bg-neutral-10 pal" style="height: 5em">Tall</div></div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">&nbsp;</div>
+ </div>
+ <div class="fitem-xs mbxl">
+  <div class="bg-neutral-10">align end</div>
+ </div>
+</div>
+
+*/
+
+.frow {
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-left: -(floor($grid-gutter-width / 2));
+  margin-right: -(ceil($grid-gutter-width / 2));
+}
+
+.fcol {
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.frow.reverse {
+  flex-direction: row-reverse;
+}
+
+.fcol.reverse {
+  flex-direction: column-reverse;
+}
+
+@mixin flex-base {
+  box-sizing: border-box;
+  flex: 0 0 auto;
+  .frow>& { // Only horizontally do we make gutters
```

Comment:
Hmm. `*=` would be dangerous: that'd match `not-a-fitem` too. I think that might be getting too clever. I'd love to reduce the size of the emitted code, but the actual complexity is low.